docs/source/examples/supplychain/overview.rst

```diff
@@ -0,0 +1,100 @@
+*******************
+SupplyChain Example
+*******************
+
+In this example provided the infrastructure necessary track physical objects
+thru a real world supply chain on the Sawtooth Ledger. Obviously we can not put
```

Comment:
Reword similar to the Overview section of the other document ( sawtooth-core/docs/source/examples/supplychain/supplychain_transaction_family.rst )
for sense and flow.